src/app/rundown/components/rundown-header/rundown-header.component.html

```diff
@@ -0,0 +1,46 @@
+<div [cdkContextMenuTriggerFor]="context_menu" class="c-header" [class.c-header--active]="rundown.isActive">
+    <ng-template #context_menu>
+    <sofie-rundown-header-context-menu class="c-context-access-surface" [rundown]="rundown"> </sofie-rundown-header-context-menu>
+    </ng-template>
+
+    <div class="c-information-row">
+        <div class="c-information-row-left">
```

Comment:
When naming css classes, we should aim to make descriptive names that describe their purpose, not how they are visually displayed.

Reply:
This entire structure has now been changed.